draft-boro-opsawg-teas-attachment-circuit.md

```diff
@@ -84,6 +84,11 @@ The meanings of the symbols in the YANG tree diagrams are defined in {{?RFC8340}
 
 This document uses the following terms:
 
+Bearer:
```

Comment:
"bearer" is a actually a key definition. We should double-focus on this one imho. We might have a call together just for this part.

A few comments meanwhile:
- I would use a different word than attachment (since we use attachment circuit): maybe connection to a customer site.
- you say "(they cover layer 1 and Layer 2)". I think the "and", is confusing since people may think L2 config (e.G. a vlan) is mandated.  This is more a "or", but we should actually phrase differently.
- add example (i.e. identifier of phyiscial connection)